index.html

```diff
@@ -513,29 +513,140 @@ <h3>Assert Claim</h3>
       </dl>
     </section>
     <section>
-      <h3>Verify Claim</h3>
+      <h3>Verify Credential</h3>
       <dl class="dl-horizontal">
         <dt>Requirement</dt>
         <dd>
-It MUST be possible for a <a>verifier</a> to verify that the credential is an
-authentic statement of an <a>issuer's</a> <a>claims</a> about the
-<a>subject</a>. The verifying <a>entity</a> must have the capability to
-connect the issuer’s identity to its credential identifier and the
-<a>subject's</a> identity to their identifier as indicated in the credential.
-The issuer’s verification information, such as its public key, must be
-discoverable from the credential record and verifiably linked to the issuer.
-It MUST be possible to do this in an automated fashion.
+          It MUST be possible for a <a>verifier</a> to verify that the
+          credential is an authentic statement of an <a>issuer's</a>
+          <a>claims</a> about the
+          <a>subject</a>. Verifiers must have the capability to
+          cryptographically prove that a credential has not been tampered
+          with since issuance (authenticity) and that the issuer continues
+          to assert the claims as true (currency). 
+          
+          To check authenticity, the issuer's verification methods, such as
+          a public key, must be discoverable from the credential itself. It
+          is expected that the issuer identifier either is
+          cryptographically verifiable itself or that it can be used to
+          reliably discover the cryptographic material necessary to confirm
+          that the content of a credential has not been changed since
+          issuance. It is understood that verification depends on the
+          verifier's acceptance of the robustness of the cryptographic
+          mechanisms used for testing authenticity and for discovering
+          verification methods. An untrusted verification method cannot be
+          relied upon for verification.
+
+          To check the currency of a credential, verifiers must be able to
+          inquire about the current status of a credential without
+          revealing to the issuer 
+          <ol type="a">
+             <li>Who is requesting status</li>
+             <li>Which credential is being checked</li>
+             <li>The subject of the credential being checked</li> 
+          </ol>
+          This status check allows issuers to revoke, suspend, or otherwise
```

Comment:
following the last section "this status check" makes me think the act of "checking" allows "issuers to revoke, suspend ..."

perhaps rephrase to something like

"This status capability..."

Reply:
```suggestion
          Requiring that verifiers perform this status check is what enables issuers to revoke, suspend, or otherwise
```